vendor/ddcloud/resource_vip_pool_member.go

```diff
@@ -45,10 +45,9 @@ func resourceVIPPoolMember() *schema.Resource {
 				Computed: true,
 			},
 			resourceKeyVIPPoolMemberPort: &schema.Schema{
-				Type:     schema.TypeString,
```

Comment:
Hi  - can we talk about this change? It looks like a reduction in functionality (as well as being a breaking change).

Currently, you can specify the string "ANY" in place of a port number, and it will match any port. If you change this to an integer, how would you express a pool membership that's not tied to a specific port?